scanner/src/main/kotlin/ScanResultsStorage.kt

```diff
@@ -152,29 +152,39 @@ abstract class ScanResultsStorage {
                 "Password for PostgreSQL storage is missing."
             }
 
-            val properties = Properties()
-            properties["user"] = config.username
-            properties["password"] = config.password
-            properties["ApplicationName"] = "$ORT_FULL_NAME - $TOOL_NAME"
-
-            // Configure SSL, see: https://jdbc.postgresql.org/documentation/head/connect.html
-            // Note that the "ssl" property is only a fallback in case "sslmode" is not used. Since we always set
-            // "sslmode", "ssl" is not required.
-            properties["sslmode"] = config.sslmode
-            config.sslcert?.let { properties["sslcert"] = it }
-            config.sslkey?.let { properties["sslkey"] = it }
-            config.sslrootcert?.let { properties["sslrootcert"] = it }
-
-            val connection = DriverManager.getConnection(config.url, properties)
+            val dataSourceConfig = HikariConfig().apply {
+                jdbcUrl = config.url
+                username = config.username
+                password = config.password
+                // use a value slightly bigger than the number of threads accessing the storage
```

Comment:
Nits: Start capitalized, end with a dot. How high is "the number of threads accessing the storage", and where is it configured?

Reply:
Changed and referred to the LocalScanner.scanPackages() function. I do not want to repeat the actual number here, as this would introduce redundancy.

Reply:
So how about extracting the `5` to a constant in the companion object of `LocalScanner`, and then using something like `LocalScanner.NUM_STORAGE_THREADS + 3` here?

Reply:
See https://github.com/oss-review-toolkit/ort/pull/3456.